### ISSUE TITLE
How to pre-train noise stream's vgg weights on ImageNet?

### ISSUE THREAD

**Issue by yisampi** [LarryJiang134/Image_manipulation_detection#8]
hello, can you talk about the pre-train noise stream's vgg weights on ImageNet more details?
thanks.
for accurate prediction, please pre-train noise stream's vgg weights on ImageNet and overwrite the trainable setting of noise stream after SRM conv layer

**Comment by andylulu** [comment]
Yes,i encounteered the problem too. In Image_manipulation_detection-master/data/imagenet_weights/ 
 the file vgg16.ckpt  how to pre-train to generate my own model parameters?Thanks a lot!

**Comment by zqdeepbluesky** [comment]
I met the same question

**Comment by JakubReha** [comment]
Did anyone solve it?

**Comment by duanxingjian** [comment]
"for accurate prediction, please pre-train noise stream's vgg weights on ImageNet and overwrite..." read it over and over but absolutely no clue what that means. What ImageNet? Overwrite how?
